tests/collections/tts/test_tts_exportables.py

```diff
@@ -15,8 +15,10 @@
 import tempfile
 
 import pytest
+from omegaconf import DictConfig, OmegaConf
```

Comment:
nit: `DictConfig` is an unused import. Please remove.

Reply:
Done.